### DISCUSSION TITLE
Help: Open Office/Libre Office both not working, Guessing due to homebrew.

### DISCUSSION THREAD

**Discussion by zoiii111** [Homebrew/discussions#2892] [Getting started]
Mac 2021 (Big Sur): 

Cannot open neither Open Office nor Libre Office. Both get killed immediately. Tried de- and reinstallation, did not work. 
I (IT-newbie) stupidly installed homebrew and guess, this is causing problems. Uninstalling homebrew does not work:
My Input: 
~ % /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/uninstall.sh)"

Getting this: 
mkdir: /tmp/uninstall.69122: Permission denied
Unable to create temp dir '/tmp/uninstall.69122'

Tried installing open Office from here: 
~ % brew install --cask openoffice

Got me this: 
Error: Permission denied @ rb_sysopen - /private/tmp/github_api_headers20220131-69606-e7h269

Grateful for any help :)

Thank u in advance!!!

**Comment by neutric** [comment]
Your Homebrew installation is broken. This report lacks context necessary to analyze what might be going wrong, though. Please file an [issue](https://github.com/Homebrew/homebrew-cask/issues/new?assignees=&labels=&template=01_bug_report.yml), which will guide you regarding necessary information to provide.

**Comment by gromgit** [comment]
What's the output of `ls -lLd /tmp`?

**Reply by zoiii111** [reply]
drwxr-xr-x@ 4 root  wheel  128 Feb  2 05:54 /tmp

**Reply by carlocab** [reply]
Try
```
HOMEBREW_TEMP="$TMPDIR" brew install --cask openoffice
```

**Reply by gromgit** [reply]
> drwxr-xr-x@ 4 root wheel 128 Feb 2 05:54 /tmp

That's not right, `/tmp` should be world-writable _and_ sticky:
```
$ ls -lLd /tmp
drwxrwxrwt 7 root wheel 224 Feb  2 21:16 /tmp/
```
`sudo chmod 1777 /tmp/` and try your install again.

**Reply by zoiii111** [reply]
Is there any possibility for me to just uninstall homebrew all together? 
I tried the protocol provided by hombrew but it always says: Permission denied...

**Reply by zoiii111** [reply]
> HOMEBREW_TEMP="$TMPDIR" brew install --cask openoffice

Output:  
Error: Permission denied @ rb_sysopen - /private/tmp/github_api_headers20220202-89058-dutlig

**Reply by SMillerDev** [reply]
> Is there any possibility for me to just uninstall homebrew all together?
I tried the protocol provided by hombrew but it always says: Permission denied...

Yeah, because the permissions for your OS are messed up.

Try the suggestion above because until you fix those permissions nothing is going to work correctly.
